### PR TITLE
storm syntax additions and csv importer

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/lib/cmd.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        self.outp = outp
+        self.exited = False
+
+        argparse.ArgumentParser.__init__(self,
+                                         prog=prog,
+                                         formatter_class=argparse.RawDescriptionHelpFormatter,
+                                         **kwargs)
+
+    def exit(self, status=0, message=None):
```
Comment (vEpiphyte) [inline synapse/lib/cmd.py]: this *has* to raise an exception.

File: synapse/lib/ast.py [hunk 2]
```diff
@@ -255,17 +317,92 @@ class VarSetOper(Oper):
...
+class VarListSetOper(Oper):
+
+    async def run(self, runt, genr):
+
+        names = self.kids[0].value()
+
+        if self.kids[1].isRuntSafe():
+
+            item = self.kids[1].runtval(runt)
+            if len(item) < len(names):
```
[the remark this answers is not in the document]
Comment (invisig0th) [reply synapse/lib/ast.py]: yep.  you broke it, you bought it. :)

File: synapse/lib/cmd.py (hunk 1 above)
Comment (vEpiphyte) [reply synapse/lib/cmd.py]: fixed

File: synapse/tools/feed.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import os
 import sys
 import time
+import types
 import shutil
 import logging
 import argparse
```
[the remark this answers is not in the document]
Comment (vEpiphyte) [reply synapse/tools/feed.py]: fixed

**Pull request merged**